apps/patient/src/views/Pharmacy.tsx

```diff
@@ -611,9 +616,18 @@ export const Pharmacy = () => {
         <VStack spacing={4} align="span" py={4}>
           {location ? (
             <VStack spacing={9} align="stretch">
-              {enableMailOrder ? (
+              {enableCourier || enableMailOrder ? (
                 <BrandedOptions
-                  options={orgSettings.mailOrderNavigateProviders ?? []}
+                  options={[
+                    ...(enableCourier && order?.address?.postalCode
+                      ? [
+                          capsuleZipcodeLookup[
+                            order.address.postalCode as keyof typeof capsuleZipcodeLookup
+                          ]?.pharmacyId
```

Comment:
If the postal code is not in the lookup, you might end up with an `undefined` in there. I dont think this is possible though?

Reply:
I would probably do something like

```ts
const capsulePharmacy = capsuleZipcodeLookup[
                            order.address.postalCode as keyof typeof capsuleZipcodeLookup
                          ]?.pharmacyId;
```
And then 
```tsx
options={[...(enableCourier && order?.address?.postalCode && capsulePharmacy ? [capsulePharmacy] : []) ...)
```

Still kinda ugly but I think a bit better

Reply:
You can also do:
```ts
const options = [
    enableCourier && order?.address?.postalCode && capsulePharmacy, 
    enableMailOrder && orgSettings.mailOrderNavigateProviders
].filter(x => x)
```

Reply:
updated